eodag/resources/providers.yml

```diff
@@ -2913,9 +2913,9 @@
           - '{$.completionTimeFromAscendingNode#get_hydrological_year}'
     SATELLITE_CARBON_DIOXIDE:
       dataset: satellite-carbon-dioxide
-      processing_level: level_2
+      processing_level: level_3
```

Comment:
In their download UI there is no day, month, year selection anymore if level_3 is selected. Did you test if the date filter is still working (it is still in the metadata mapping).

Reply:
I works for level 2 but for level 3 it has no effect. Is it an acceptable behavior?